hack/jenkins/test-history/gen_json.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates a JSON file containing test history for the last day.
+
+Writes the JSON out to tests.json.
+"""
+
+from __future__ import print_function
+
+import json
+import os
+import re
+import subprocess
+import sys
+import time
+import urllib2
+import xml.etree
+import zlib
+
+
+def get_json(url):
+    """Does an HTTP GET to url and parses the JSON response. None on failure."""
+    try:
+        content = urllib2.urlopen(url).read().decode('utf-8')
+        return json.loads(content)
+    except urllib2.HTTPError:
+        return None
+
+def get_jobs(server):
+    """Generates all job names running on the server."""
+    jenkins_json = get_json('{}/api/json'.format(server))
+    if jenkins_json is None:
```

Comment:
Favor implicit false (if not jenkins_json)

https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations